sentry-core/src/performance.rs

```diff
@@ -774,13 +774,15 @@ impl Transaction {
     }
 
     /// Returns the headers needed for distributed tracing.
+    /// Use [`crate::Scope::iter_trace_propagation_headers`] to obtain the active
+    /// span's/transaction's distributed tracing headers.
```

Comment:
The idea is that this could also return the ids from PropagationContext if there is no ongoing span, not sure if this comment correctly reflects that. 
But on the other hand we should also hide PropagationContext as that's an "implementation detail".

Reply:
In that case, I would suggest rephrasing this comment as follows:

```suggestion
    /// Use [`crate::Scope::iter_trace_propagation_headers`] to obtain the active
    /// trace's distributed tracing headers.
```

This is technically correct, and you avoid going into detail about span/transaction/propagation context/etc.

You may want to update the other comments as well